packages/autocomplete-plugin-algolia-insights/src/createAlgoliaInsightsPlugin.ts

```diff
@@ -137,6 +145,10 @@ export function createAlgoliaInsightsPlugin(
     return {};
   }
 
+  if (insightsInitParams) {
+    insightsClient('init', { partial: true, ...insightsInitParams });
+  }
```

Comment:
Should we add a condition that checks we can use the partial init here (`isModernInsightsClient()`) ?

Reply:
as it's additional, and only when people pass the insights init parameters, I'm not sure if it's needed